modules/stanford_news_views/README.md

```diff
@@ -16,6 +16,3 @@ The 5 items news list page is identical to the 5 item list block except for that
 
 * **3 Item news list block with page**   r
 The three item news list block displays exactly three of the most recent news items in a formatted list. The list contains a thumbnail, title, date, tags, and short description. 
-
```

Comment:
Upgrade path is here:
https://github.com/SU-SWS/stanford_news/blob/7.x-3.x-dev/stanford_news.install